pyVHDLParser/Blocks/InterfaceObject.py

```diff
@@ -258,7 +258,7 @@ def stateSubtypeIndication(cls, parserState: TokenToBlockParser):
 			parserState.Counter =       0
 			return
 		elif isinstance(token, CharacterToken):
-			if token == '(':
+			if token == '(' or token == ';':
```

Comment:
Please split changes to tokens from changes to blocks.

Reply:
Done. I removed the change from this PR and will include it in a future PR.